src/mmdata.c

```diff
@@ -3067,16 +3066,12 @@ long **alloc2DMatrix(size_t xsize, size_t ysize)
   long **matrix;
   long i;
   matrix = malloc(xsize * sizeof(long *));
-  if (matrix == NULL) {
-    fprintf(stderr,"?FATAL ERROR 1376 Out of memory\n");
-    exit(1);
-  }
+  if (matrix == NULL)
+    exitOnFatalError(__FILE__, __LINE__, "?FATAL ERROR 1376 Out of memory\n");
```

Comment:
Are these (1376, and below, 1377) magic numbers ? Or do they correspond to something ?

Reply:
The numbers are markers identifying the particular reason for a bug or an error.  A pair (__FILE__, __LINE__) could serve more or less the same idea.  This is the usual way to do, and it s part of the interface in my software here.